docs/src/components/ComponentDoc/ComponentProp/ComponentPropFunctionSignature.tsx

```diff
@@ -3,7 +3,16 @@ import PropTypes from 'prop-types'
 import * as React from 'react'
 
 import { neverUpdate } from 'docs/src/hoc'
-import ComponentPropExtra from './ComponentPropExtra'
+import ComponentPropExtra, { ComponentPropExtraProps } from './ComponentPropExtra'
+
+interface ComponentPropFunctionArg extends ComponentPropExtraProps {
```

Comment:
This definitely should be `ComponentPropFunctionProps`

Reply:
done, thanks